pandas/io/sql.py

```diff
@@ -1421,7 +1421,13 @@ def run_transaction(self):
 
     def execute(self, *args, **kwargs):
         """Simple passthrough to SQLAlchemy connectable"""
-        return self.connectable.execution_options().execute(*args, **kwargs)
+        if "chunksize" in kwargs:
```

Comment:
if you are going to do this, then chunksize should be an actual keyword with a default, and a doc-string.

are there *any* tests which actually evaluate this path?